tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ManagedIdentityTests.NetFwk.cs

```diff
@@ -123,6 +123,61 @@ public async Task AcquireMSITokenAsync(MsiAzureResource azureResource, string us
             }
         }
 
+        [TestMethod]
+        public async Task AcquireMSITokenForTokenExchangeResourceAsync()
```

Comment:
See my general suggestions for test naming: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4694#discussion_r1566325119

I would maybe rename to this for easier reading:
```suggestion
        public async Task AcquireMsiToken_ForTokenExchangeResource_Successfully()
```